rest-api/src/main/java/gov/cms/qpp/conversion/api/services/DbServiceImpl.java

```diff
@@ -110,7 +113,7 @@ public Metadata getMetadataById(String uuid) {
 	 */
 	@Override
 	protected Metadata asynchronousAction(Metadata meta) {
-		mapper.save(meta);
+		mapper.get().save(meta);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Call "mapper.isPresent()" before accessing the value. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3655)